### DISCUSSION TITLE
OpenGraph Compatible Meaning

### DISCUSSION THREAD

**Discussion by countrymusicfy** [AzuraCast/AzuraCast#4225] [Q&A] [answered]
"The embeddable data on each station's public player is now 'OpenGraph' compatible, which should work across multiple social media platforms, including the existing Twitter player implementation."

Does this mean that a station's public player can now post on Facebook?  If so, how?

Otherwise, what is an example of 'OpenGraph' compatible across multiple social media platforms?

**Comment by BusterNeece** [comment, marked as answer]
@countrymusicfy This means we now use `og:` metadata tags on the public player page's HTML headers, which are indexed and used by all major social media sites per the universal Open Graph standards: https://ogp.me/

This does indeed mean that you can link your station via Facebook and it should pull some of that metadata and provide a richer experience, but unfortunately the only site we're aware of that offers a built-in radio player is Twitter, which requires proprietary tags to accomplish that purpose.
